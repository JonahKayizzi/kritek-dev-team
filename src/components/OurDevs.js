import React from 'react';
import DevForm from './DevForm';

const OurDevs = () => {
  const addDevItem = () => {
    //
  };

  let content;

  return (
    <main>
      <DevForm addDevProp={addDevItem} />
      {content}
    </main>
  );
};

export default OurDevs;
