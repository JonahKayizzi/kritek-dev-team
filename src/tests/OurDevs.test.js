import { render } from '@testing-library/react';
import OurDevs from '../components/OurDevs';

describe('OurDevs tests', () => {
  test('Snapshot test. Should render OurDevs correctly', () => {
    const tree = render(<OurDevs />);
    expect(tree).toMatchSnapshot();
  });
});
