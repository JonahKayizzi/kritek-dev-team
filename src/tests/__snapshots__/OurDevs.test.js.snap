// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurDevs tests Snapshot test. Should render OurDevs correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul
        class="flex flex-col gap-5 py-4 bg-gray-50 px-24 shadow-lg shadow-slate-300"
      >
        <form
          class="border-b mb-1 pb-5 flex flex-row flex-wrap gap-4"
        >
          <p
            class="text-gray-400 text-xl font-semibold tracking-wide text-left w-full"
          >
            ADD NEW DEV
          </p>
          <input
            class="w-1/5 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="name"
            placeholder="Name ..."
            type="text"
            value=""
          />
          <select
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="specialty"
          >
            <option
              value=""
            >
              Specialty
            </option>
            <option
              value="Full Stack"
            >
              Full Stack
            </option>
            <option
              value="Frontend"
            >
              Frontend
            </option>
            <option
              value="Backend"
            >
              Backend
            </option>
            <option
              value="Mobile Development"
            >
              Mobile Development
            </option>
          </select>
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="position"
            placeholder="Position ..."
            type="text"
            value=""
          />
          <select
            class="w-1/5 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="type"
          >
            <option
              value=""
            >
              Employment Type
            </option>
            <option
              value="Freelance"
            >
              Freelance
            </option>
            <option
              value="Full Time"
            >
              Full Time
            </option>
            <option
              value="Part Time"
            >
              Part Time
            </option>
            <option
              value="Contract"
            >
              Contract
            </option>
          </select>
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="location"
            placeholder="Location"
            type="text"
            value=""
          />
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="salary"
            placeholder="Salary"
            type="text"
            value=""
          />
          <input
            class="w-1/6 text-xs border border-gray-300 rounded-sm py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="dob"
            placeholder="Date of Birth"
            type="date"
            value=""
          />
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="contact"
            placeholder="Contact"
            type="text"
            value=""
          />
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="email"
            placeholder="Email"
            type="text"
            value=""
          />
          <input
            class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
            name="photo"
            placeholder="Photo Link"
            type="text"
            value=""
          />
          <button
            class="font-semibold bg-cyan-500 text-white rounded-sm py-2 px-3 hover:bg-cyan-600 focus:outline-none focus:shadow-outline"
            type="submit"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-plus "
              data-icon="plus"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
                fill="currentColor"
              />
            </svg>
          </button>
        </form>
        <li
          class="flex flex-col items-center"
        >
          <div
            class="go1471829640 "
          />
        </li>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul
      class="flex flex-col gap-5 py-4 bg-gray-50 px-24 shadow-lg shadow-slate-300"
    >
      <form
        class="border-b mb-1 pb-5 flex flex-row flex-wrap gap-4"
      >
        <p
          class="text-gray-400 text-xl font-semibold tracking-wide text-left w-full"
        >
          ADD NEW DEV
        </p>
        <input
          class="w-1/5 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="name"
          placeholder="Name ..."
          type="text"
          value=""
        />
        <select
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="specialty"
        >
          <option
            value=""
          >
            Specialty
          </option>
          <option
            value="Full Stack"
          >
            Full Stack
          </option>
          <option
            value="Frontend"
          >
            Frontend
          </option>
          <option
            value="Backend"
          >
            Backend
          </option>
          <option
            value="Mobile Development"
          >
            Mobile Development
          </option>
        </select>
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="position"
          placeholder="Position ..."
          type="text"
          value=""
        />
        <select
          class="w-1/5 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="type"
        >
          <option
            value=""
          >
            Employment Type
          </option>
          <option
            value="Freelance"
          >
            Freelance
          </option>
          <option
            value="Full Time"
          >
            Full Time
          </option>
          <option
            value="Part Time"
          >
            Part Time
          </option>
          <option
            value="Contract"
          >
            Contract
          </option>
        </select>
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="location"
          placeholder="Location"
          type="text"
          value=""
        />
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="salary"
          placeholder="Salary"
          type="text"
          value=""
        />
        <input
          class="w-1/6 text-xs border border-gray-300 rounded-sm py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="dob"
          placeholder="Date of Birth"
          type="date"
          value=""
        />
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="contact"
          placeholder="Contact"
          type="text"
          value=""
        />
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="email"
          placeholder="Email"
          type="text"
          value=""
        />
        <input
          class="w-1/6 border border-gray-300 rounded-sm py-3 py-2 px-2 text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
          name="photo"
          placeholder="Photo Link"
          type="text"
          value=""
        />
        <button
          class="font-semibold bg-cyan-500 text-white rounded-sm py-2 px-3 hover:bg-cyan-600 focus:outline-none focus:shadow-outline"
          type="submit"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-plus "
            data-icon="plus"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
              fill="currentColor"
            />
          </svg>
        </button>
      </form>
      <li
        class="flex flex-col items-center"
      >
        <div
          class="go1471829640 "
        />
      </li>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
